Hb raci vEN for Public Cloud Instances

H4516022Y opened the pull request:
hi,

This a translation in EN of RACI already published in french

Thanks in advance for the proofreading and actions following.

Happy holidays and best wishes for the new year

H4516022Y committed:
PCI RACI vEN

To be translated

H4516022Y committed:
Update guide.en-en.md

H4516022Y committed:
Update guide.en-en.md

ovh-cds commented:
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image img_2903.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2900.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2901.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2902.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Other images have empty alt attributes
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/ak-as-ca.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/sequence01.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/validate-ck-ca.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/sequence02.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: More {attach} warnings silenced.
WARNING: Unable to find `images/3734.png`, skipping url replacement.
WARNING: Unable to find `images/3735.png`, skipping url replacement.
WARNING: Unable to find `images/3736.png`, skipping url replacement.
WARNING: Unable to find `images/3738.png`, skipping url replacement.
WARNING: Other resources were not found and their urls not replaced
CRITICAL: TemplateNotFound: menus/menu.en-en.html
```

H4516022Y commented:
hi,
I don't understand what's the blocking point.
It's the first time I see such error

H4516022Y committed:
EN RACI for Instances

new PR due to the error in the last one : https://github.com/ovh/docs/pull/3930
I don't understand what's the blocking point on it so I created a new one
could you help please ?
the doc 'guide.en-en.md' could be deleted

ovh-cds commented:
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image 3948.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3949.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3936.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3953.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Other images have empty alt attributes
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/ak-as-ca.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/sequence01.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/validate-ck-ca.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: The {attach} link in account/api/api_right_delegation/guide.en-ca.md cannot relocate account/api/api_right_delegation/images/sequence02.png because its output location was already overridden. Falling back to {filename} link behavior instead.
WARNING: More {attach} warnings silenced.
WARNING: Unable to find `images/3734.png`, skipping url replacement.
WARNING: Unable to find `images/3735.png`, skipping url replacement.
WARNING: Unable to find `images/3736.png`, skipping url replacement.
WARNING: Unable to find `images/3738.png`, skipping url replacement.
WARNING: Other resources were not found and their urls not replaced
CRITICAL: TemplateNotFound: menus/menu.en-en.html
```

OVH-Team-Guides commented:
Hello @H4516022Y, it's because the "guide.en-en.md" is not an accepted variable. Please delete this file to only keep the "guide.en-gb.md" one, that should fix this issue.

H4516022Y committed:
Delete guide.en-en.md

H4516022Y committed:
Update guide.en-gb.md

H4516022Y committed:
Update guide.en-gb.md

Y0Coss committed:
Proofreading EN RACI for Public Cloud Instances

OVH-Team-Guides commented:
Thank you for your contribution @H4516022Y
